### PR TITLE
Subaru: add ability to send infotainment status message

### PR DESCRIPTION
**Pull request by jnewb1**
see https://github.com/commaai/opendbc/pull/809

### CODE REVIEW

**Comment by adeebshihadeh** [comment]
What's this for? Are you planning on opening a matching openpilot PR?

**Comment by jnewb1** [comment]
> What's this for? Are you planning on opening a matching openpilot PR?

Yes. This is for filtering the little message that shows up on the Subaru infotainment system because eyesight long is still running in the background, and the steering wheel is not being turned.

**Review by adeebshihadeh — COMMENTED** [review]
Looks good, two remaining things TODO:
- [ ] update the subaru unit test: https://github.com/commaai/panda/blob/878e0077ac077c5325b37710b8df856468d85674/tests/safety/test_subaru.py#L10
- [ ] revert the change to board/obj/.placeholder

Then, we can merge once the openpilot side is ready.

File: board/safety/safety_subaru.h [hunk 1]
```diff
@@ -24,15 +24,17 @@ const CanMsg SUBARU_TX_MSGS[] = {
   {0x122, 0, 8},
   {0x221, 0, 8},
   {0x321, 0, 8},
-  {0x322, 0, 8}
+  {0x322, 0, 8},
+  {0x323, 0, 8},
```
Comment (sshane) [inline board/safety/safety_subaru.h:28]: remove from global non-gen2 for now

File: tests/safety/test_subaru.py [hunk 2]
```diff
@@ -7,11 +7,11 @@
 
 
 class TestSubaruSafety(common.PandaSafetyTest, common.DriverTorqueSteeringSafetyTest):
-  TX_MSGS = [[0x122, 0], [0x221, 0], [0x321, 0], [0x322, 0]]
+  TX_MSGS = [[0x122, 0], [0x221, 0], [0x321, 0], [0x322, 0], [0x323, 0]]
```
Comment (sshane) [inline tests/safety/test_subaru.py:10]: same

Comment (jnewb1) [reply tests/safety/test_subaru.py]: done

File: board/safety/safety_subaru.h (hunk 1 above)
Comment (jnewb1) [reply board/safety/safety_subaru.h]: done

**Pull request merged**